### PR TITLE
More code cleanups

### PR DESCRIPTION
**Pull request by rfk**
This addresses a couple of nits we had left over from #71.  @chilts r?

### CODE REVIEW

File: email_record.js [hunk 1]
```diff
@@ -115,26 +115,27 @@ module.exports = function (RATE_LIMIT_INTERVAL_MS, BLOCK_INTERVAL_MS, MAX_EMAILS
...
-
-    if (this.isOverEmailLimit()) {
-      // rate limit this email if now over the limit and tell them to retry
-      this.rateLimit()
-      return this.retryAfter()
+    // For email-sending actions, we may need to rate-limit.
+    if (isEmailSendingAction(action)) {
+      // If they're already being blocked then don't count any more hits,
+      // and tell them to retry.
+      if (this.shouldBlock()) {
```
Comment (shane-tomlinson) [inline email_record.js]: What is the difference between `shouldBlock` and `isBlocked`?

Comment (rfk) [inline email_record.js]: It's...complicated.  "should block" == "is explicited marked as blocked" || "has exceeded sending rate limit".  We could probably name these better.

**Comment by rfk** [comment]
Bump for @chilts to take a look at this please, let's make some incremental cleanups in this repo over time.

File: email_record.js (hunk 1 above)
Comment (chilts) [inline email_record.js]: +10101 for renaming these things as we see fit. And also, get rid of those pesky really short prefixes in the objects, but that's a rant for another day. :)

**Comment by chilts** [comment]
This all looks fine, mostly mechanical (1st and 3rd commit). Second commit looks cool too. r+

**Pull request merged**